src/component/organisms/index-indicator.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { SetStateAction, useEffect } from 'react';
+import styled, { css } from 'styled-components';
+import { useRouter } from 'next/router';
+
+import ChevronLeftIcon from '@src/assets/icons/common/chevron/left';
+import ChevronRightIcon from '@src/assets/icons/common/chevron/right';
+import { Row, Clickable, P } from '@src/component/atoms';
+import { BLACK0, GREY0, WHITE0 } from '@src/component/atoms/colors';
+
+import { constructUrl, getQueryParameters } from '@src/lib/utils/Url';
+
+import { prefetchItem } from '@src/lib/api';
+
+type ListIndexIndicatorProps = {
+  index: number;
+  count: number;
+  pageSize: number;
+  setPage?: React.Dispatch<SetStateAction<number>>;
+  requestConfig: any;
+  filter: any;
+  routing: boolean;
+};
+
+const PAGES_PER_VIEW = 5;
+
+export default function ListIndexIndicator({
+  index,
+  count,
+  pageSize,
+  setPage,
+  requestConfig,
+  filter,
+  routing,
+}: ListIndexIndicatorProps) {
+  const router = useRouter();
+
+  useEffect(() => {
+    if (index > 0) {
+      prefetchItem(
+        requestConfig({
+          offset: (index - 1) * pageSize,
+          limit: pageSize,
+          ...filter,
+        })
+      );
+    }
+    prefetchItem(
+      requestConfig({
+        offset: (index + 1) * pageSize,
+        limit: pageSize,
+        ...filter,
+      })
+    );
+  });
+
+  const handlePageMove = (page) => {
+    const pathname = window.location.pathname;
+    const href = window.location.href;
+
+    setPage(page);
+    if (!routing) return;
+
+    window.scrollTo(0, 0);
+    router.push(
+      constructUrl(pathname, { ...getQueryParameters(href), page: page + 1 })
+    );
+  };
+
+  const getIndexButtons = (index, startIndex, endIndex) => {
+    return [...Array(endIndex - startIndex)].map((_v, i) => {
+      const nowIndex = startIndex + i;
+      const selected = nowIndex === index;
+      return (
+        <I onClick={() => handlePageMove(nowIndex)} selected={selected} key={i}>
+          <IndexText selected={selected}>{nowIndex + 1}</IndexText>
+        </I>
+      );
+    });
+  };
+
+  if (Math.floor(count / (pageSize + 1)) !== 0) {
+    const startIndex = index - (index % PAGES_PER_VIEW);
+    const endIndex = Math.min(
+      startIndex + PAGES_PER_VIEW,
+      Math.floor(count / (pageSize + 1)) + 1
+    );
+    const havePrev = startIndex !== 0;
+    const haveNext = endIndex < Math.floor(count / (pageSize + 1)) + 1;
+    return (
+      <IndicatorWrapper>
```

Comment:
```suggestion
      <Wrapper>
```

Wrapper라고만 네이밍해줘도 될 것 같아요!

Reply:
0ca24d4